.devcontainer/devcontainer.json

```diff
@@ -22,8 +22,9 @@
     // Add the IDs of extensions you want installed when the container is created.
     "extensions": [
         "dbaeumer.vscode-eslint",
+        "editorconfig.editorconfig",
         "ms-azuretools.vscode-docker",
-        "editorconfig.editorconfig"
+        "mutantdino.resourcemonitor"
     ],
```

Comment:
(Orthogonal: Sneaking in a nice Resource Monitor ext to VS Code when running in a Dev Container.)